pydoctor/astbuilder.py

```diff
@@ -48,7 +48,7 @@ def _handleAliasing(
         ) -> bool:
     """If the given expression is a name assigned to a target that is not yet
     in use, create an alias.
-    @return: L{True} iff an alias was created.
+    @return: L{True} if an alias was created.
```

Comment:
Did the semantics of this somehow change so that it is not if-and-only-if? (If you were trying to correct a spelling error, "iff" is in fact a word: https://en.wiktionary.org/wiki/iff )